### PR TITLE
Fix CSS issues

### LINKED ISSUE #431: Padding lost on documentation homepage
Somehow the padding got lost here, making the page look slightly broken:

bildschirmfoto 2017-07-06 um 11 14 00

I assume that wasn't by intention?

### PR DESCRIPTION
**Pull request by ThomDietrich**
* margins in lists
* code padding
* code line-height
* overall content margin-right, closes #431

Signed-off-by: Thomas Dietrich <Thomas.Dietrich@tu-ilmenau.de>

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -733,6 +735,11 @@ body.documentation .container {
     width: auto;
 }
 
+body.documentation section#documentation .container {
+    margin-left: 10px;
+    margin-right: 20px;
```
Comment (kubawolanin) [inline css/styles.css]: Do we follow any formatting guideline in this file?
There seem to be properties with just one space on the left and here we have four.

Comment (kubawolanin) [reply css/styles.css]: http://csscomb.com/ ;-)

Comment (ThomDietrich) [reply css/styles.css]: Totally aware of that 😅  I never dared to unleash an autoformatter on that file.

Just so you know: These are some quick fixes to make the documentation presentable now. I'll present a proposal for a new design soon...

**Comment by ThomDietrich** [comment]
The added changes to fix #431 introduced a new issue on wide screens. I'll look at it later today. @kubawolanin feel free to propose something **if** you are free

**Pull request merged**